src/services/bria/index.ts

```diff
@@ -104,14 +104,14 @@ export const BriaSubscriber = () => {
 
 const queueNameForSpeed = (speed: PayoutSpeed): string => briaConfig.queueNames[speed]
 
-export const NewOnChainService = (): INewOnChainService => {
+export const OnChainService = (): IOnChainService => {
```

Comment:
I don't think we have really used the `I` prefix consistently. I think we may have been moving away for that (which is fine - type are in a different domain so they could be spell the same)

Reply:
I haven't really changed any convention here, just used what was previously being used in this module - going through the whole codebase and trying to improve consistency on this particular topic is out of scope of this PR.